volunteer_planner/settings/sentry.py

```diff
@@ -0,0 +1,43 @@
+"""
+Imports and initializes sentry for usage within this Django instance.
+
+For successful initialization 'sentry-sdk' needs to be installed and some
+environment variables need to be present/set:
+- SENTRY_DSN: sentry data source name (https://docs.sentry.io/product/sentry-basics/dsn-explainer/)  # noqa: E501
+              mandatory, without no initialization will be successfully finished.
+- SENTRY_ENVIRON: string, used as 'environment' tag value
+                  default: 'production'
+- SENTRY_TRACE_RATE: float, (0..1), probability a transaction is performance monitored
+                     default: 0.0
+- SENTRY_SEND_PII: bool, (https://docs.sentry.io/platforms/python/configuration/options/#send-default-pii)  # noqa: 501
+                   default: False
+"""
+import logging
+import os
+
+from distutils.util import strtobool
+
+logger = logging.getLogger(__name__)
+
+try:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    from sentry_sdk.integrations.logging import LoggingIntegration
+except (ModuleNotFoundError, ImportError):
+    logger.warning("sentry not installed - skipping", exc_info=True)
+
+try:
+    sentry_sdk.init(
```

Comment:
Move into try block in above try...except handling, so it would not be executed in the first place, when importing failed with an exception?

Reply:
Wasn't sure, if this construct would be OK.
Thanks, fixed.